apps/journeys-admin/src/components/JourneyList/RadioSelect/RadioSelect.tsx

```diff
@@ -0,0 +1,227 @@
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
+  const breakpoints = useBreakpoints()
+  const triggerRef = useRef<HTMLDivElement>(null)
+
+  const currentValue = value ?? defaultValue
+  const currentLabel =
+    options.find((option) => option.value === currentValue)?.label ?? ''
+
+  useEffect(() => {
+    setAnchorEl(triggerRef.current)
+  }, [triggerRef])
+
+  const handleClick = (event: MouseEvent<HTMLElement>): void => {
+    setAnchorEl(event.currentTarget)
+    setShowOptions(true)
+  }
+
+  const handleKeyDown = (event: KeyboardEvent<HTMLElement>): void => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      event.preventDefault()
+      setAnchorEl(event.currentTarget)
+      setShowOptions(true)
+    }
+  }
+
+  const handleClose = (): void => {
+    setAnchorEl(null)
+    setShowOptions(false)
+  }
+
+  const handleSubmit = (event: ChangeEvent<HTMLInputElement>): void => {
+    event.preventDefault()
+    handleChange(event.currentTarget.value as T)
+    handleClose()
+  }
+
+  const Form = (): ReactElement => (
+    <Box>
+      <FormControl component="fieldset" fullWidth>
+        <RadioGroup
+          aria-label={`${ariaLabel}-options`}
+          defaultValue={currentValue}
+          name={`${ariaLabel}-buttons-group`}
+          onChange={handleSubmit}
+          sx={{
+            gap: 0,
+            '& .MuiFormControlLabel-root': {
+              margin: 0
+            }
+          }}
+        >
+          {options.map((option) => (
+            <FormControlLabel
+              key={option.value}
+              value={option.value}
+              control={
+                <Radio
+                  size="small"
+                  sx={{
+                    height: '20px',
+                    width: '20px',
+                    marginRight: '10px'
+                  }}
+                />
+              }
+              label={option.label}
+              sx={{
+                padding: '8px 20px 8px 12px',
+                '&:hover': {
+                  backgroundColor: 'rgba(0, 0, 0, 0.1)'
+                },
+                '& .MuiFormControlLabel-label': {
+                  fontFamily: "'Open Sans', sans-serif",
+                  fontWeight: 400,
+                  fontSize: '14px',
+                  lineHeight: '22px'
+                }
+              }}
+            />
+          ))}
+        </RadioGroup>
+      </FormControl>
+    </Box>
+  )
+
+  const showMobileIcon = !breakpoints.sm && mobileIcon != null
+
+  return (
+    <>
+      <Box
+        ref={triggerRef}
+        onClick={handleClick}
+        onKeyDown={handleKeyDown}
+        role="button"
+        aria-label={ariaLabel}
+        aria-haspopup="listbox"
+        aria-expanded={showOptions}
+        sx={{
+          display: 'inline-flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          borderRadius: '8px',
+          border: '2px solid',
+          borderColor: palette[700],
```

Comment:
Replace with 'secondary.light'

```suggestion
          borderColor: 'secondary.light',
```